kolibri/core/assets/src/views/CoreMenu/__tests__/CoreMenuOption.spec.js

```diff
@@ -0,0 +1,202 @@
+import { render, fireEvent, screen } from '@testing-library/vue';
+import userEvent from '@testing-library/user-event';
+import VueRouter from 'vue-router';
+import CoreMenuOption from '../CoreMenuOption.vue';
+import '@testing-library/jest-dom';
```

Comment:
I think we can now remove this import, as we already have it in the jest `setup.js` file.

Reply:
I tried that, but it didn't work. Removing the import for `userEvent` started giving a linting error, as well as an error when running the test suite (saying that `userEvent` is not defined).

Reply:
Oh I was talking about the `import '@testing-library/jest-dom';` :sweat_smile: but it seems that you already deleted it :hugs: